GlotPress/GlotPress-WP
# Add plural and plural forms labels to row previews and format the row editor accordingly

## What?
Add plurals to original in the translation row preview, and add plural forms labels, accordingly to translate.wp.org

## Why?
When filtering the Untranslated strings, there is no visual information about plural strings, either on original or translation column.
Example:
![imagem](https://user-images.githubusercontent.com/7371591/194959715-d8a46bb8-18ff-4080-877e-95b0a2bbeef8.png)

If the translation is added, only the translation shows the plural forms.
![imagem](https://user-images.githubusercontent.com/7371591/194959939-88c974c4-a915-402e-90e4-7a53cb5ffd07.png)


## How?
This PR adds the same logic to the template as the [template on wp.org](https://github.com/WordPress/wordpress.org/blob/f63aa20a0b8ad126d758046c155f148fb0e6d6c2/wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/templates/translation-row-preview.php), implementing this:
- [x] Add plural to original plural strings
- [x] Add Singular/Plural and other plural forms (e.g. Russian) to both original and translation texts
- [x] Vertical align to center the row plurals

Result for Singular/Plural (e.g.: Portuguese):
![imagem](https://user-images.githubusercontent.com/7371591/194960812-ef872ccd-6a67-4f26-8493-d8fe613a13a3.png)

Result for other plural form (e.g.: Russian):
![imagem](https://user-images.githubusercontent.com/7371591/194960903-4c084346-3ce4-4ce4-9521-60cbd2b64d78.png)

## Testing Instructions

1. Filter plural strings
2. See row preview with and without translation
3. Check it on languages with Singular/Plural and languages with othet plural forms (eg. Russian translation set)

Fixes #1505
